site/pages/view-all-disruptions.page.tsx

```diff
@@ -588,7 +703,7 @@ export const getServerSideProps = async (): Promise<{ props: ViewAllDisruptionsP
                 status: Progress.open,
                 severity: getWorstSeverity(severitys),
                 serviceIds,
-                operators,
+                operators: disruptionOperators,
```

Comment:
Unable to find where this field is used

Reply:
It is used in the page on line 267 - operators.

Reply:
the const `operators` from line 659 is used in line 267 as that is what is returned as props in 718 and 728

Reply:
Ah ok - the operators: disruptionOperators is what is returned inside the map function, so its the operators which are associated with a disruption.

Reply:
so is `operators: disruptionOperators` required? If its not then line 720-731 could be redundant?

Reply:
yeah its required otherwise we dont know which operators are in a disruption